bindings/python/pinocchio/deprecated.py

```diff
@@ -195,3 +195,42 @@ def log6FromSE3(transform):
 @deprecated("This function will be removed in future releases of Pinocchio. You can use log or log6.")
 def log6FromMatrix(matrix4):
   return pin.log6(matrix4)
+
+@deprecated("This function has been renamed getJointJacobian and will be removed in future releases of Pinocchio. Please change for new getJointJacobian function.")
+def getJacobian(model,data,jointId,local):
+  if local:
+    return pin.getJointJacobian(model,data,jointId,pin.ReferenceFrame.LOCAL)
+  else:
+    return pin.getJointJacobian(model,data,jointId,pin.ReferenceFrame.WORLD)
+
+# This function is only deprecated when using a specific signature. Therefore, it needs special care
+# Marked as deprecated on 16 Sept 2019
+def impactDynamics(model, data, q = None, *args):
```

Comment:
Uhm... I think there are some problems... This way, Python calls such as `impactDynamics(model,data,v_before,J[,...])` will not work...

Reply:
Seems working well.